src/components/Button.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+function Button({name,onClick,disabled}){
+
+    return(
+<>
+        <button className="button" name={name} type="button" onClick={onClick} disabled={disabled}>
+            {name}
+        </button>
+</>
+
+    )
+
+}
```

Comment:
There is a lot of overhead with these small components. Why don't use <button /> directly without this wrapper?